backend/db/analysis.py

```diff
@@ -165,43 +165,37 @@ def counts_over_time(
 
     # Pivot
     values = 'count' if method == 'counts_table' else 'delta'
-    data_df = current_counts_df.pivot(index='table', columns='timestamp', values=values).fillna(0).astype(int)
+    df = current_counts_df.pivot(index='table', columns='timestamp', values=values).fillna(0).astype(int)
 
-    # Add note
-    with get_db_connection(schema='', local=local) as con:
-        runs = [dict(x) for x in sql_query(
-            con, f"SELECT timestamp, note FROM counts_runs WHERE schema = '{schema}';", return_with_keys=True)]
-    timestamps = [x['timestamp'] for x in runs]
-    ts_dict = {}
-    for ts in timestamps:
-        for run in runs:
-            if run['timestamp'] == ts:
-                ts_dict[ts] = run['note']
-    runs_df = pd.DataFrame([ts_dict])
-    runs_df = runs_df.reindex(sorted(runs_df.columns), axis=1)
-    runs_df.index = ['COMMENT']
-    df = pd.concat([runs_df, data_df])
+    dateslist = [column[:10] for column in df.columns]
+    dates = list(set(dateslist))
+
+    finaldf = pd.DataFrame()
+    for date in dates:
+        # Create temporary table with the columns of only one date
+        datedf = df[df.columns[df.columns.str.startswith(date)]]
+        datedf = datedf.sort_index(axis=1)
+        # todo: List number of times the refresh ran that day somewhere?
+        # count = dateslist.count(date)
+        # datedf.loc[f'Number of refreshes ran'] = count
```

Comment:
@hugwuoke I just renamed the new special rows "delta/count over time, 1x/day" to "Number of refreshes ran", but then commented it out to remove for now. Showing the number of refreshes that ran that day, which is what this was doing, may be potentially useful, but I think better to leave it off the table for now.

I was wondering why you added this here, and then I read my [initial instructions](https://github.com/jhu-bids/TermHub/issues/418#issue-1766522253) and saw that they were entirely unclear / misstated!:
>  Add two new table sections, "Deltas over time, 1x/day", and "Counts over time, 1x/day".

What I actually meant was to add new sections to the markdown file; i.e. in addition to the 2 tables in the `analysis.md` that have 1 column per run, we could have 2 more tables that aggregate these per day. But I think that what you've done is actually better; just having 1 table for counts and 1 table for deltas, and having those be aggregated to 1x/day. 

I think what we have here is good. I think no need for the weekly aggregation right now. I'll merge this and let that issue be closed. Thanks so much!